docs/index.rst

```diff
@@ -132,7 +132,7 @@ Data is transmitted from the phone to the server (and vis-a-versa) over a secure
     :caption: Code
     :maxdepth: 1
 
-    toggles
+    toggles/toggles
```

Comment:
Is that naming going to cause any issues in future? We have the same for both the levels.

Reply:
If we ever add a subfolder in toggles called toggles then this might cause an issue (although I don't feel that scenario is very likely). When adding this, I was following the design pattern we used for messaging where it is also `messaging/messaging` for the index page. I'm happy to change this if you think it would be better with a name such as `index`

Reply:
how about `toggles/index`?

Reply:
I've renamed the file for both `toggles` and `messaging` to `index` for consistency ([2b70e6f](https://github.com/dimagi/commcare-hq/pull/34548/commits/2b70e6fdb3d1da7a59352c9702c8d269214debb8)).

Reply:
Brilliant, thank you.